R/REST_helpers.R

```diff
@@ -36,12 +36,12 @@ sbtools_POST <- function(url, body, session, timeout = default_timeout(), ...){
 #' @keywords internal
 sbtools_GET <- function(url, ..., session, timeout = default_timeout()){
 	
-	supported_types <- c('text/plain','application/json')
+	supported_types <- c('text/plain','application/json','application/x-gzip')
```

Comment:
What about providing an argument for accepted types and setting the default to what we have now? I also like @jread-usgs's idea to check for changes in the url, though i wonder if that covers all error cases. @lawinslow, yeah, the SB folks might have good ideas - you willing to check with them?